skeletonkey/instantiate.py

```diff
@@ -24,143 +28,151 @@ def import_target(class_string: str) -> Type[Any]:
     return obj
 
 
-def instantiate_partial(config, target_keyword=TARGET_KEYWORD,_instantiate_recursive=True, **kwargs) -> Any:
-    """
-    Instantiate a functools.partial object using a Config object. The Config
-    object should contain the key "_target_" to specify the class to instantiate.
-
-    You need to call the partial object with the remaining parameters to
-    fully instantiate.
+def instantiate(*configs:Iterable[Config], 
```

Comment:
Not proper styling for skeletonkey. Check later comment.

Reply:
Fixed